lib/constants/constants.go

```diff
@@ -246,6 +248,9 @@ const (
 	ETCDKeyPair = "etcd"
 	// OpsCenterKeyPair is a name of key pair for OpsCenter
 	OpsCenterKeyPair = "ops"
+	// PlanetRPCKeyPair is a keypair for planet's RPC client for
+	// satellite monitoring and exchange
+	PlanetRpcKeyPair = "planet-rpc-client"
```

Comment:
PlanetRpcKeyPair -> PlanetRPCKeyPair

Reply:
This will also be the cert/key filename, won't it? How about we just call it "planet"? So it'll be planet.cert / planet.key. Or "satellite".

Reply:
Well, the difficulty is, this doesn't really distinguish if this is a client key or a server key if we just call it planet or satellite.

Right now we don't generate these key's separately, so it's not a big deal, but I think something we'll need to tackle at some point is actually setting ClientKeyUsage/ServerKeyUsage based on the key usage, which means this key isn't really the planet server key, it's the client key. This is why I was headed the direction of marking the client key in it's name.